src/app/analytics/analytics.factory.js

```diff
@@ -537,9 +537,15 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
               category: cartItem.designationType.toLowerCase(),
               variant: cartItem.frequency.toLowerCase(),
               quantity: '1',
-              dimension3: cartItem.frequency.toLowerCase() === 'single' ? 'one-time' : 'recurring',
+              dimension1: localStorage.getItem('gaDonorType'),
+              dimension2: localStorage.getItem('gaDonorAcct') || undefined,
+              dimension3: cartItem.designationType.toLowerCase(),
               dimension4: cartItem.frequency.toLowerCase(),
-              dimension8: cartItem.designationType.toLowerCase()
+              dimension5: cartItem.frequency.toLowerCase() === 'single' ? 'one-time' : 'recurring',
+              dimension6: purchaseData.paymentMeans['account-type'] ? purchaseData.paymentMeans['account-type'].toLowerCase() : purchaseData.paymentMeans['card-type'].toLowerCase(),
+              dimension7: purchaseData.rawData['purchase-number'],
+              dimension8: 'designation',
```

Comment:
Any references in the codebase to ```Siebel Product Type``` was always set to ```designation``` so I did the same thing here.

Reply:
Yeah, there are other types, but we don't currently support them in the give site. In the future we might pull in others.